<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="2" skipped="3" tests="37" time="1.031" timestamp="2023-03-16T16:03:59.613045" hostname="BAN-LAP-VREDDY"><testcase classname="Tests.test_abcd" name="test_addition" time="0.002" /><testcase classname="Tests.test_abcd" name="test_sub" time="0.001" /><testcase classname="Tests.test_abcd" name="test_multi" time="0.001" /><testcase classname="Tests.test_abcd" name="test_division" time="0.001" /><testcase classname="Tests.test_basic_fixture" name="test_comparewithAA" time="0.001" /><testcase classname="Tests.test_basic_fixture" name="test_comparewithBB" time="0.001" /><testcase classname="Tests.test_basic_fixture" name="test_comparewithCC" time="0.002" /><testcase classname="Tests.test_circle.Test_Circle" name="test_radius" time="0.001"><error message="failed on setup with &quot;file D:\Workspace\Practice\project1\Tests\test_circle.py, line 7&#10;      def test_radius(self, radius):&#10;E       fixture 'radius' not found&#10;&gt;       available fixtures: _verify_url, base_url, browser, browser_channel, browser_context_args, browser_name, browser_type, browser_type_launch_args, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, context, delete_output_dir, device, doctest_namespace, extra, include_metadata_in_junit_xml, input_value, is_chromium, is_firefox, is_webkit, launch_browser, metadata, monkeypatch, page, playwright, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id&#10;&gt;       use 'pytest --fixtures [testpath]' for help on them.&#10;&#10;D:\Workspace\Practice\project1\Tests\test_circle.py:7&quot;">file D:\Workspace\Practice\project1\Tests\test_circle.py, line 7
      def test_radius(self, radius):
E       fixture 'radius' not found
&gt;       available fixtures: _verify_url, base_url, browser, browser_channel, browser_context_args, browser_name, browser_type, browser_type_launch_args, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, context, delete_output_dir, device, doctest_namespace, extra, include_metadata_in_junit_xml, input_value, is_chromium, is_firefox, is_webkit, launch_browser, metadata, monkeypatch, page, playwright, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, testrun_uid, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, worker_id
&gt;       use 'pytest --fixtures [testpath]' for help on them.

D:\Workspace\Practice\project1\Tests\test_circle.py:7</error></testcase><testcase classname="Tests.test_circle.Test_Circle" name="test_area" time="0.001" /><testcase classname="Tests.test_compare" name="test_greater" time="0.001"><skipped type="pytest.xfail" message="" /></testcase><testcase classname="Tests.test_compare" name="test_greater_equal" time="0.001" /><testcase classname="Tests.test_compare" name="test_less" time="0.001"><skipped type="pytest.skip" message="unconditional skip">D:\Workspace\Practice\project1\Tests\test_compare.py:18: unconditional skip</skipped></testcase><testcase classname="Tests.test_depend" name="test_a" time="0.001"><skipped type="pytest.xfail" message="deliberate fail" /></testcase><testcase classname="Tests.test_depend" name="test_b" time="0.001" /><testcase classname="Tests.test_depend" name="test_c" time="0.001" /><testcase classname="Tests.test_depend" name="test_d" time="0.001" /><testcase classname="Tests.test_depend" name="test_e" time="0.001" /><testcase classname="Tests.test_div_by_13" name="test_divisible_by_13" time="0.002" /><testcase classname="Tests.test_div_by_3_6" name="test_divisible_by_3" time="0.001" /><testcase classname="Tests.test_div_by_3_6" name="test_divisible_by_6" time="0.001"><failure message="assert (39 % 12) == 1">input_value = 39

    def test_divisible_by_6(input_value):
&gt;       assert input_value % 12 == 1
E       assert (39 % 12) == 1

test_div_by_3_6.py:8: AssertionError</failure></testcase><testcase classname="Tests.test_failure" name="test_sqrt_failure" time="0.001" /><testcase classname="Tests.test_failure" name="test_square_failure" time="0.001" /><testcase classname="Tests.test_failure" name="test_equality_failure" time="0.001" /><testcase classname="Tests.test_letsdo" name="test_class2" time="0.001" /><testcase classname="Tests.test_multi_11" name="test_multiplication_11[1-11]" time="0.001" /><testcase classname="Tests.test_multi_11" name="test_multiplication_11[2-22]" time="0.003" /><testcase classname="Tests.test_multi_11" name="test_multiplication_11[3-33]" time="0.001" /><testcase classname="Tests.test_multi_11" name="test_multiplication_11[4-44]" time="0.001" /><testcase classname="Tests.test_some" name="test_class2" time="0.001" /><testcase classname="Tests.test_square" name="test_sqrt" time="0.001" /><testcase classname="Tests.test_square" name="testsquare" time="0.001" /><testcase classname="Tests.test_square" name="test_equality" time="0.001" /><testcase classname="Tests.test_upper" name="test_capital_case" time="0.001" /><testcase classname="Tests.test_wallet" name="test_func1" time="0.001" /><testcase classname="Tests.test_wallet" name="test_func2" time="0.002"><failure message="assert False">def test_func2():
&gt;       assert False
E       assert False

test_wallet.py:11: AssertionError</failure></testcase><testcase classname="Tests.test_welcome" name="test_wel" time="0.001" /><testcase classname="Tests.test_yield_fix" name="test_context_fixture_order" time="0.006" /></testsuite></testsuites>